tools/recetox_aplcms/macros.xml

```diff
@@ -146,6 +145,25 @@
                    help="The minimum number of raw data points to be considered as a true feature." />
         </section>
     </xml>
+    <xml name="multibatch_processing">
+        <section name="multibatch_processing" title="Multibatch processing">
+            <param name="min_within_batch_prop_detect" type="float" min="0" max="1" value="0.1"
+                   label="minimum_batchwise_prop_detect"
+                   help="The minimum proportion of samples in a batch that must have a given feature. The features that are less abundant than the value won't be extracted." />
+            <param name="min_within_batch_prop_report" type="float" min="0" max="1" value="0.5"
+                   label="minimum_batchwise_prop_report"
+                   help="The minimum proportion of samples in a batch that must have a given feature. The features that are less abundant than the value won't be reported in the final output." />
+            <param name="min_batch_prop" type="float" min="0" max="1" value="0.5"
+                   label="minimum_batch_prop"
+                   help="The minimum proportion of batches that must have a given feature. The features that are less abundant than the value won't be reported." />
+            <param name="batch_align_mz_tol" type="float" min="0" value="0.00001"
+                   label="batch_align_mz_tol"
+                   help="The m/z tolerance level for peak alignment within batch." />
+            <param name="batch_align_chr_tol" type="float" min="0" value="50.0"
+                   label="batch_align_chr_tol"
+                   help="The retention time tolerance level for peak alignment within batch." />

```

Comment:
Are there extra parameters for alignment across batches or do they use the same values?

Reply:
The Two-Step Hybrid has distinct parameters for aligning _m/z_ and _rt_ between batches and within batches.